From 6e27dbb798e2b4ae43c2d0f98f16e6e307c57341 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:51:05 -0400
Subject: [tools] perf/annotate: Make symbol__inc_addr_samples private

Message-id: <1408953210-26343-178-git-send-email-jolsa@redhat.com>
Patchwork-id: 88849
O-Subject: [PATCH RHEL7.1 BZ1133083 177/322] perf annotate: Make symbol__inc_addr_samples private
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 44e8303944cd45788abd48f14d4c683331ed6cf7
Author: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Wed Dec 18 17:12:24 2013 -0300

description
===========
Since it is now accessed just thru addr_map_symbol and hist_entry
wrappers.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/annotate.c b/tools/perf/util/annotate.c
index 91e2526..27ab7b5 100644
--- a/tools/perf/util/annotate.c
+++ b/tools/perf/util/annotate.c
@@ -486,8 +486,8 @@ static int __symbol__inc_addr_samples(struct symbol *sym, struct map *map,
  return 0;
 }
 
-int symbol__inc_addr_samples(struct symbol *sym, struct map *map,
-        int evidx, u64 addr)
+static int symbol__inc_addr_samples(struct symbol *sym, struct map *map,
+        int evidx, u64 addr)
 {
  struct annotation *notes;
 
diff --git a/tools/perf/util/annotate.h b/tools/perf/util/annotate.h
index 43727a9..b2aef59 100644
--- a/tools/perf/util/annotate.h
+++ b/tools/perf/util/annotate.h
@@ -132,9 +132,6 @@ static inline struct annotation *symbol__annotation(struct symbol *sym)
  return &a->annotation;
 }
 
-int symbol__inc_addr_samples(struct symbol *sym, struct map *map,
-        int evidx, u64 addr);
-
 int addr_map_symbol__inc_samples(struct addr_map_symbol *ams, int evidx);
 
 int hist_entry__inc_addr_samples(struct hist_entry *he, int evidx, u64 addr);
-- 
1.7.1